src/components/skeletons/Shimmer/shimmer.scss

```
@@ -0,0 +1,24 @@
+[data-store-shimmer] {
+  /* parent: position relative & overflow hidden */
+  position: absolute;
+
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  animation: loading 1.2s infinite;
+
+  [data-shimmer] {
+    width: 50%;
+    height: 100%;
+    background: rgb(255 255 255 / 20%);
```

```suggestion
    background: var(--color-white-transparent-20);
```
@hellofanny just notice that the `--color-white-transparent-20` is equals to `--color-black-transparent-20`. Should we change there to use white value? like `rgb(255 255 255 / 20%)`